tests/puppeteer/campaigns/sanity/02_productsBO/02_CRUDStandardProductInBO.js

```diff
@@ -2,12 +2,10 @@ require('module-alias/register');
 // Using chai
 const {expect} = require('chai');
 const helper = require('@utils/helpers');
+const initBasicPages = require('@utils/initPages');
```

Comment:
Bad naming, it's not only basicPages but whole pages, this should be called initPages

```suggestion
const initPages = require('@utils/initPages');
```